src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/update.cache.when.304.response.has.matching.etag/accept/client.rpt

```diff
@@ -29,6 +29,7 @@ write nukleus:begin.ext ${http:headers()
                               .item(":scheme", "http")
                               .item(":method", "GET")
                               .item(":path", "/data")
+                              .item("accept-encoding", "gzip")
```

Comment:
Recommend using a header other than `content-encoding: gzip` since the body of the response in the spec test is not actually gzipped.
Suggest using `content-type: text/plain` instead.

Reply:
Note: a corresponding request header would be `accept: text/plain`.